playground/eShopLite/CatalogDb/Program.cs

```diff
@@ -1,9 +1,19 @@
 using CatalogDb;
+using Microsoft.EntityFrameworkCore;
+using Npgsql;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
-builder.AddNpgsqlDbContext<CatalogDbContext>("catalogdb");
+
+var connectionString = builder.Configuration.GetConnectionString("catalogdb")
+    ?? throw new InvalidOperationException("Connection string is not configured.");
+
+// a workaround for https://github.com/npgsql/efcore.pg/issues/2821
+var configureNpgsqlLogging = (NpgsqlDataSourceBuilder builder) => { builder.UseLoggerFactory(null); };
```

Comment:
This is unfortunate...

```suggestion
var configureNpgsqlLogging = (NpgsqlDataSourceBuilder builder) => builder.UseLoggerFactory(null);
```